platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,10 +1,19 @@
 #import "MGLSource.h"
+#import "MGLTypes.h"
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface MGLRasterSource : MGLSource
 
 @property (nonatomic, readonly, copy) NSURL *URL;
 @property (nonatomic, readonly, assign) CGFloat tileSize;
```

Comment:
f7a4427 changes this to an `NSUInteger`. I think the tile size here represents "tile units" as discussed in https://github.com/mapbox/mapbox-gl-native/issues/6236